vmware_exporter/vmware_exporter.py

```diff
@@ -32,6 +33,8 @@
 from prometheus_client.core import GaugeMetricFamily
 from prometheus_client import CollectorRegistry, generate_latest
 
+from .helpers import batch_fetch_properties
```

Comment:
I saw your comment about this here --> https://github.com/pryorda/vmware_exporter/commit/3e04e22ef4721662fd30f51fde32254972287427#r31764273

But this currently fails when building/running with the provided `Dockerfile`:
```
Dec 22 17:27:47 prometheus-02 docker-vmware-exporter[5458]: Traceback (most recent call last):
Dec 22 17:27:47 prometheus-02 docker-vmware-exporter[5458]: File "/opt/vmware_exporter/vmware_exporter/vmware_exporter.py", line 36, in <module>
Dec 22 17:27:47 prometheus-02 docker-vmware-exporter[5458]: from .helpers import batch_fetch_properties
Dec 22 17:27:47 prometheus-02 docker-vmware-exporter[5458]: ModuleNotFoundError: No module named '__main__.helpers'; '__main__' is not a package
```

I tried adding the below - but it didn't help.
```
ENV PYTHONPATH "${PYTONPATH}:/opt/vmware_exporter/vmware_exporter"
```

Can you suggest a fix to `Dockerfile` to make it work with this notation?

Reply:
Hmmm I hadn't reviewed the Dockerfile yet! The problem is that its not running vmware_exporter like its a python package, its running it like a standalone script. So the way in my branch is right for a pip based installation but not a Docker based installation! I have done `pip install -e .` in my development virtualenv so it had been working fine for me.

There are 2 options - and you were very close to one of them. I think this should work: 

```
ENV PYTHONPATH "${PYTONPATH}:/opt/vmware_exporter"
```

I would also consider changing the pip install invocation to `pip install -r requirements.txt .`. This will run the setup.py installation, and we'll get a `/usr/local/bin/vmware_exporter` in the container so will need a different entrypoint:

```
CMD ["/usr/local/bin/vmware_exporter"]
```

This is probably the most useful approach as it means the Docker route shares more with the pip install route.

Reply:
I've applied the 2nd approach to my branch.